components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -310,11 +312,92 @@ std::vector<mojom::FeedItemV2Ptr> GenerateChannelBlock(
   }
 
   std::vector<mojom::FeedItemV2Ptr> result;
-  result.push_back(mojom::FeedItemV2::NewCluster(
-      mojom::Cluster::New("channel", channel, std::move(article_elements))));
+  result.push_back(mojom::FeedItemV2::NewCluster(mojom::Cluster::New(
+      mojom::ClusterType::CHANNEL, channel, std::move(article_elements))));
   return result;
 }
 
+// Generate a Topic Cluster block
+// https://docs.google.com/document/d/1bSVHunwmcHwyQTpa3ab4KRbGbgNQ3ym_GHvONnrBypg/edit#heading=h.4vwmn4vmf2tq
+std::vector<mojom::FeedItemV2Ptr> GenerateTopicBlock(
+    std::vector<Topic>& topics,
```

Comment:
nit:

> When ordering function parameters, put all input-only parameters before any output parameters. In particular, do not add new parameters to the end of the function just because they are new; place new input-only parameters before the output parameters. This is not a hard-and-fast rule. Parameters that are both input and output muddy the waters, and, as always, consistency with related functions may require you to bend the rule. Variadic functions may also require unusual parameter ordering.

https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

Reply:
Thanks! That's good to know, and seems sensible